libraries/radwordsprocessing/concepts/fields/fields.md

```diff
@@ -103,9 +116,25 @@ Here is a list of the field types that support updating:
 
 If the field type is not one of the above, the result will not be updated and the Field property of the FieldInfo class will be set to an instance of a CustomCodeField. The complete list of field codes and the switches for each of them can be found in the [Docx specification](http://www.ecma-international.org/publications/standards/Ecma-376.htm).
 
+### Updating PageRef, Page, Section, NumPages, and SectionPage fields. 
```

Comment:
Updating the Section field doesn't require calculating the size of the document elements as well as the NumberingFieldsProvider (you can check the SectionField.GetResult method). I added it for completeness.